prime_factor/prime_factor.c

```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
+#include <limits.h>
+
+int generate_primes(int num) {
+    int cnt = 0;
+    printf("Prime Factors: ");
```

Comment:
Where i store the numbers. If i store it in an array them i need to pass the size. and there is no structure such as vector in c.